radis/lbl/broadening.py

```diff
@@ -1570,7 +1677,12 @@ def _init_w_axis(w_dat, log_p):
                     lineshape = voigt_lineshape(
                         wbroad_centered, wL[m] / 2, wV_ij / 2, jit=jit
                     )  # FWHM > HWHM
+                   
+                    if self.params.chi_correction == True:
+                        lineshape *= self._add_CO2_chiCorrection(self.input.molecule, wbroad_centered, wV_ij / 2) # CO2 chi-correction 
```

Comment:
we should add this only if the molecule is CO2. 

I can take this one if you want @pgokulak